### DISCUSSION TITLE
Is there any way to cause a redirect using Apollo Server?

### DISCUSSION THREAD

**Discussion by giorgosera** [apollographql/apollo-server#6981] [Q&A]
I'm moving all my non-graphql endpoints to GraphQL using Apollo.
One of these endpoints requires a server side redirect instead of returning data to the client.
Is there any chance I can do that using Apollo Server?

**Comment by glasser** [comment]
I'm not sure what you mean by that. While Apollo Server does have a simple batteries-including HTTP server (currently the `apollo-server` package, will have a clearer name of startStandaloneServer in the nearly-released AS v4) it's also easily embeddable into your favorite web framework such as Express, and it's easy to implement such redirects in your web framework. The job of Apollo Server is processing GraphQL, not doing arbitrary HTTP things — that's what your web framework is for.

**Reply by giorgosera** [reply]
Yeah I realized that and used apollo express at the end. Thank you!
